pkg/query/conv.go

```diff
@@ -1,10 +1,56 @@
 package query
 
 import (
+	"strconv"
+
 	sitter "github.com/smacker/go-tree-sitter"
 	"go.lsp.dev/protocol"
 )
 
+func Unquote(input []byte, n *sitter.Node) string {
+done:
+	for {
+		switch n.Type() {
+		case NodeTypeModule,
+			NodeTypeBlock,
+			NodeTypeExpressionStatement:
+			n = n.Child(0)
+		case NodeTypeString:
+			break done
+		default:
+			return ""
+		}
+	}
+
+	startDelim := n.Child(0)
+	endDelim := n.Child(int(n.ChildCount() - 1))
+	byteoffset := startDelim.EndByte()
+	bytes := []byte{}
+
+	for i := 1; i < int(n.ChildCount()-1); i++ {
+		escape := n.Child(i)
+		if byteoffset < escape.StartByte() {
+			bytes = append(bytes, input[byteoffset:escape.StartByte()]...)
+			byteoffset = escape.EndByte()
+		}
+		escseq := string(input[escape.StartByte():escape.EndByte()])
```

Comment:
fwiw, I think this is just `escape.Content(input)`

Reply:
Indeed